testpmd/hooks/pre-run.yml

```diff
@@ -115,7 +115,7 @@
   git:
     repo: "https://github.com/rh-nfv-int/nfv-example-cnf-deploy.git"
     dest: "{{ dci_config_dir }}/hooks/{{ cluster_name }}/nfv-example-cnf-deploy"
-    version: "{{ example_cnf_deploy_script_version|default('master') }}"
+    version: "{{ example_cnf_deploy_script_version|default('test_event_api') }}"
```

Comment:
this should be reset to master right?

Reply:
yes, once https://github.com/rh-nfv-int/nfv-example-cnf-deploy/pull/30 is merged